pyfarm/agent/service.py

```diff
@@ -84,18 +87,19 @@ def __init__(self):
         self.services = {}
         self.register_shutdown_events = False
         self.last_free_ram_post = time.time()
-        self.shutting_down = False
```

Comment:
This is used in at least one more place (assign.py). Simply removing this breaks assign.